design-system/src/pattern-library/foundations/date-input.html

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+body-page-class: dm-dark
+title: Date input
+id: pattern-library
+---
+<article>
+    <div class="coop-l-container">
+{% include breadcrumbs.html %}
+{% for designPattern in site.data.contentful.spaces.design-system-content.designPattern %}
+    {% if designPattern.title == 'Date input' %}
+    <header>
+        <div class="coop-l-grid">
+            <div class="coop-l-grid__item coop-l-grid__item--medium-7 intro-box">
+                <h1>{{ designPattern.title }}
+                    <span class="status status--{{ designPattern.status }}">
+                        <a href="/process.html" class="status__link">{{ designPattern.status }}</a>
+                    </span>
+                </h1>
+                <div class="highlight-banner">
+                    <p class="ds-intro">{{ designPattern.introduction | markdownify }}</p>
+                </div>
+            </div>
+        </div>
+    </header>
+</div>
+<div class="fixedsticky fixedsticky--horz">
+    <nav class="coop-l-container">
+        <ul class="in-page-nav in-page-nav--horz">
+            <li><a href="#example">Example</a></li>
+            <!-- <li><a href="#design-and-content">Design and content</a></li>
```

Comment:
We have these links commented out, but the content is still visible; Could they be uncommented?